### ISSUE TITLE
Create plan for piloting project plan protoype

### ISSUE THREAD

**Issue by MelissaBraxton** [18F/TLC-crew#364]
### A description of the work
This issue covers drafting a plan for piloting the [prototype project planning template](https://airtable.com/appdlhWUaEISJEYZa/tblmVgWHVxl1qTMan/viwruD6WRWpuSbmvN?blocks=hide) created in #331. The pilot plan should include:
 -  what we hope to learn from the pilot
 - any necessary preconditions
 - who to include in the pilot (teams, project leads, account managers?) and participation criteria (e.g., project phase, project duration?)
 - how long the pilot should run
 -  how we'll invite participation in the pilot 
 - how to communicate to participants what they should expect

### Point of contact on this issue
@MelissaBraxton and @alexbielen 

### Reproduction steps (if necessary)
Be as specific as possible

**Billable?**

- [ ] Yes
- [X] No
If yes, tock code:

**Skills needed**

- [X] Any human
- [ ] Design
- [ ] Content
- [ ] Engineering
- [ ] Acquisition
- [ ] Product
- [ ] Other

**Timeline**
Does this need to happen in the next two weeks?

- [ ] Yes
- [X] No

How much time do you anticipate this work taking?
About a week

### Acceptance Criteria

- [ ] We have a pilot plan that includes
 -  what we hope to learn from the pilot
 - any necessary preconditions
 - who to include in the pilot (teams, project leads, account managers?) and participation criteria (e.g., project phase, project duration?)
 - how long the pilot should run
 -  how we'll invite participation in the pilot 
 - how to communicate to participants what they should expect
 - [ ] The pilot plan has been reviewed and approved by TLC crew lead(s) 
 - [ ] We have documented next steps in executing the plan as new issues.

**Comment by SelenaJV** [comment]
@christophermather and @donnykwan Re: Limited AirTable licenses constraint...Perhaps you've considered this but maybe this data can be submitted a form rather than giving more folks direct editing permissions? Maybe we modify the project health form to include questions about project plan/phase? 

Also, we have a small working group creating and improving AirTable dashboards. You should chat with them about display thoughts if you have not already. AlexS, NatashaI, and ElieB are the primary folks working on #18f-wg-airtable

**Comment by donnykwan** [comment]
Thanks @SelenaJV for the suggestion.  We did talk to NatashaJ and exchanged info  of our works.

**Comment by christophermather** [comment]
Might be a good idea to update Natasha on our progress! I'll share the table and see what she thinks. Thanks, @SelenaJV!

**Comment by jdamis4XD** [comment]
@christophermather, @donnykwan and I met with @NatashaJIbrahim and @alexsobledotgov earlier today to give context on the project and demo the Airtable prototype that was created.  We determined that the Airtable licenses required to implement this prototype would be a significant blocker for this project tracking plan to be successful.  At this time, until GMT makes some decisions we propose pausing this work.

**Comment by alexsobledotgov** [comment]
@jdamis4XD -- Hmm, your comment about the Airtable licenses doesn't fully track with what I remember from our discussion. Although Airtable licenses are limited, there are lots of ways we can allow 18F team members to add or edit data without needing more licenses. Creating Airtable forms is one way we can do this -- that's the approach we're using for getting staffing data into the system. It looks like @SelenaJV mentioned this approach above. 

That said, if your group doesn't want to move forward with the prototype, that's OK! I'm just not sure based on my experience working in Airtable that the license thing has to be a blocker.

@jdamis4XD @christophermather @donnykwan @NatashaJIbrahim please chime in if I'm missing something here 🤔 🤔

**Comment by jdamis4XD** [comment]
@alexsobledotgov Thanks for commenting.  You did propose that forms could be used as a work around to the licenses constraint for editing, but I recall we also discussed that managing and keeping track of licences was also out of our control and would require reliance on another team outside of 18F.  @christophermather, @donnykwan and debriefed after our Demo meeting to review our discussion and concurred that the licenses would ultimately be a barrier.

**Comment by christophermather** [comment]
Agreeing w/ @jdamis4XD and @donnykwan that licenses are a blocker to move this specific ticket forward. If this can't be widely adopted due to licenses and data governance concerns, there's no sense in testing and gauging interest in the prototype in its current form.

With that said, the only color I would add to this is that we haven't completely ruled out **any** type of Airtable solution as a means of moving this epic (#240) forward. An Airtable form built directly into the 18F Dashboard is under discussion, but it would be pretty different from the current prototype (and might need to be built from the ground up).

**Comment by alexsobledotgov** [comment]
@christophermather @jdamis4XD @donnykwan — FYSA, we just got word from LaKeisha in Slack that TTS will be moving to an Enterprise plan for Airtable, she hopes the completion time for this will be short — check out the #airtable-smartsheets channel.

Looks like Enterprise plans include unlimited seats and workspaces: https://airtable.com/pricing 😍 😍

**Comment by jdamis4XD** [comment]
@alexsobledotgov Thanks for the update. That would be awesome.

**Comment by MelissaBraxton** [comment]
@donnykwan  - Does this still belong in "blocked/waiting?" Is there anything @alexbielen and I need to do to help unblock it?

**Comment by jdamis4XD** [comment]
@christophermather Do you think this ticket is still blocked or should a new issue be created to close this ticket?   

cc: @donnykwan

**Comment by donnykwan** [comment]
Given that  Airtable license  is no longer a blocker, I recommend this put back to the queue of next sprint. 
@MelissaBraxton @alexbielen @jdamis4XD @christophermather

**Comment by MelissaBraxton** [comment]
@donnykwan and @christophermather:

Here's an example pilot plan you can use. It's a bit more detailed and involved than what I'd imagine y'all will need. Notably, the "objectives" section is roughly analogous to assumptions we need to test. 

https://docs.google.com/document/d/1S30BJ3wEx9hYCyfm1HZdHUgeIXdVN8I6ctTswN6BmP4/edit

**Comment by christophermather** [comment]
@donnykwan - do you mind sharing the pilot plan here? Once shared, I think we can move this to "done" and open up a new follow-on issue for initiating the pilot.

**Comment by donnykwan** [comment]
The pilot plan doc is [here](https://docs.google.com/document/d/1IIzcMNDAR536lAzYlGPEksJH1tjyEgpu1G9pHnwtAj4/edit#heading=h.mjb4baddzb96).

**Comment by donnykwan** [comment]
Moving this to Done.
